packages/@adobe/spectrum-css-temp/components/accordion/skin.css

```diff
@@ -62,3 +62,10 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Accordion-itemHeader {
+    &:focus-ring {
+      outline: 3px solid CanvasText;
```

Comment:
How is the focus style drawn for `.spectrum-Accordion-itemHeader` without WHCM? Should this be?:
```css
@media (forced-colors: active) {
  .spectrum-Accordion-itemHeader {
    &:focus-ring {
      &:after  {
        background-color: CanvasText;
      }
    }
  }
}
```

Reply:
without WHCM it is a border at the left hand side. Really too subtle for WHCM. IMO it is not enough which is why I added the outline.